### DISCUSSION TITLE
How to use pyswmm to view Surface Runoff in swmm's report file

### DISCUSSION THREAD

**Discussion by Messi0617** [pyswmm/pyswmm#443] [Q&A] [answered]
Hello, here are my questions
The surface runoff in the SWmm report file is very useful for my analysis process. How can I view the data in the report file
![图片](https://github.com/pyswmm/pyswmm/assets/135831731/efad2c4d-6c48-4015-a126-61ed16f30679)
I am eagerly looking forward to your answer, and I will be extremely grateful！

**Comment by bemcdonnell** [comment, marked as answer]
@Messi0617,

Here are a couple of links for you:

As for your request, check out the documentation for "System Stats"
http://pyswmm.github.io/pyswmm/reference/system.html#pyswmm.system.SystemStats

If you are looking for where to put the code to get the statistics out, see the "latte" example at www.pyswmm.org
https://www.pyswmm.org/examples#h.a8kx6p7pjmc3

**Reply by Messi0617** [reply]
Thank you for taking the time to answer my question online. Your expertise and willingness to help are greatly appreciated.
